src/sender.rs

```diff
@@ -343,12 +346,12 @@ where
 
     /// Send a message `content` to a single `recipient`.
     #[tracing::instrument(
-        skip(self, unidentified_access, message, recipient),
-        fields(unidentified_access = unidentified_access.is_some(), recipient = %recipient),
+        skip(self, unidentified_access, message),
+        fields(unidentified_access = unidentified_access.is_some(), recipient = recipient.service_id_string()),
```

Comment:
```suggestion
        fields(unidentified_access = unidentified_access.is_some(), recipient = %recipient.service_id_string()),
```

Reply:
This is never necessary when you deal with strings.

Reply:
Oh. Ok! :-)